alerting/src/main/kotlin/org/opensearch/alerting/DocumentReturningMonitorRunner.kt

```diff
@@ -273,6 +294,21 @@ object DocumentReturningMonitorRunner : MonitorRunner() {
         return finding.id
     }
 
+    private fun updateLastRunContext(
```

Comment:
will this require any change in the monitor schema, since we have per index context to be persisted?

Reply:
This does not require further changes to the monitor schema